splinterd/src/config/partial.rs

```diff
@@ -785,4 +821,45 @@ impl PartialConfig {
         self.metrics_password = metrics_password;
         self
     }
+
+    #[cfg(feature = "log-config")]
+    /// Adds a `verbosity` value to the `PartialConfig` object.
+    ///
+    /// # Arguments
+    ///
+    ///  * `level` - The log level the system will use for the root_logger logger
+    pub fn with_verbosity(mut self, level: Option<log::Level>) -> Self {
+        self.verbosity = level;
+        self
+    }
+    #[cfg(feature = "log-config")]
```

Comment:
Missing a newline above this line (also, same with functions below in this chunk).

Reply:
fixed all the spacing issues